ops/internal/manage/staging.go

```diff
@@ -144,6 +114,7 @@ func InflateChainConfig(sc *StagedChain) (*config.Chain, error) {
 
 	if dc.UseAltDA {
 		cfg.Addresses.DAChallengeAddress = config.NewChecksummedAddress(dc.DAChallengeProxy)
+		cfg.DataAvailabilityType = "alt-da"
```

Comment:
There are two `if dc.UseAltDA` blocks (here and line 61-68). Could they be combined?

Reply:
fixed